pom.xml

```diff
@@ -113,6 +113,11 @@
             <artifactId>workflow-cps-global-lib</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>io.jenkins.plugins</groupId>
+          <artifactId>okhttp-api</artifactId>
+          <version>4.9.2-20211102</version>
```

Comment:
this version should be included in the [bom](https://github.com/jenkinsci/bom/blob/1090.v0a_33df40457a_/bom-weekly/pom.xml#L94-L98), so I believe the version should be removed from the declaration 

```suggestion
```

Reply:
Also, isn't the okhttp-plugin is already pulled in via the github-api-plugin?

https://github.com/jenkinsci/github-api-plugin/blob/ec734c8434b39d344c3d9d7ff29041d4aeff09f3/pom.xml#L37-L41

Reply:
@bitwiseman I am not sure about that, but even if it was the case it is a best practice to declare the dependencies your project relies on directly in your POM in order to keep it transparent what your project directly relies on.